flocker/dockerplugin/_api.py

```diff
@@ -60,11 +66,29 @@ def decorator(f):
             schema_store=SCHEMAS,
             ignore_body=ignore_body)
         def wrapped(*args, **kwargs):
-            return f(*args, **kwargs)
+            d = maybeDeferred(f, *args, **kwargs)
```

Comment:
:grey_question: Potentially add to the docstring above the new functionality that this PR is adding to the decorator.